.gitignore

```diff
@@ -1,15 +1,131 @@
-*.iml
+### AndroidStudio ###
+# Covers files to be ignored for android development using Android Studio.
+
+# Built application files
+*.apk
```

Comment:
Updated the group's .gitignore files to include more files than the default .gitignore had. This helps prevent pushing or committing certain files, reducing unnecessary clutter and keep the repository clean.

https://www.atlassian.com/git/tutorials/saving-changes/gitignore
https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files

Reply:
Which additional files were added here?  Are they files that are actually present in the project?

While the .gitignore should be through, as with all source code, it should also be concise.  No need to remove files that don't exist.